synth.log

```diff
@@ -0,0 +1,1324 @@
+
+> @aws-quickstart/eks-blueprints@1.15.1 examples
```

Comment:
What is this file? Why is this added to PR?

Reply:
removed, good catch.